api/docs/test_suite.dox

```diff
@@ -315,6 +315,15 @@ The comments at the top of runsuite_ssh.cmake describe additional options.
 
```

Comment:
> iX: Record how to mark a test as flaky

Repeating the same comment from your first PR: we do not put "IX: " in titles; there is no prefix if there is no issue.

Reply:
Done, with separate PR #6586 to codify the rule.